test/commands/create_test.dart

```diff
@@ -173,7 +174,7 @@ void main() {
     );
 
     expect(projectDir.childFile('example/lib/main.dart'), exists);
-    expect(projectDir.childDirectory('example/tizen').listSync(), isNotEmpty);
+    expect(projectDir.childFile('example/tizen/Runner.csproj'), exists);
```

Comment:
I wondered if I really needed a cpp example. Since this is just an example, I think it would be right to always generate a csharp example that can accommodate both cpp and csharp plugin.